com.ktechpit.ultimate-media-downloader.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2022 Keshav Bhatt <keshavnrj@gmail.com> -->
+<component type="desktop">
+  <id>com.ktechpit.ultimate-media-downloader</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://raw.githubusercontent.com/keshavbhatt/ultimate-media-download-linux/main/LICENSE</project_license>
```

Comment:
weird, the file at the URL list a MIT license.

Reply:
Updated.

Reply:
doesn't seem to have changed.

Reply:
This is it, I mean Its my own custom License :D